src/functions/template-tags/loop.php

```diff
@@ -128,7 +128,13 @@ function tribe_get_events_title( $depth = true ) {
 
 		$tribe_ecp = Tribe__Events__Main::instance();
 
-		$title = sprintf( esc_html__( 'Upcoming %s', 'the-events-calendar' ), $events_label_plural );
+		if ( is_single() && tribe_is_event() ) {
```

Comment:
There are other single post possibilities, like venues (where we list upcoming events), was my thinking. Could certainly strip the `tribe_is_event()` check even so if you prefer?